### DISCUSSION TITLE
Can't create new database connection on Mac

### DISCUSSION THREAD

**Discussion by fishmonger1** [dbeaver/dbeaver#21212] [Q&A] [answered]
### Description

_OS- Mac, Ventura, M1 chip._ 
I downloaded the correct MySQL version (ARM) from https://dev.mysql.com/downloads/mysql/. 
The server is active:
<img width="887" alt="Screenshot 2023-09-14 at 12 12 52" src="https://github.com/dbeaver/dbeaver/assets/145003880/77ebced3-44e2-4905-bf50-b0517933affa">

When opening Dbeaver, there is no option to create a  new database connection. The button is grayed out. See screenshot:
<img width="967" alt="Screenshot 2023-09-14 at 11 59 49" src="https://github.com/dbeaver/dbeaver/assets/145003880/7c06ead7-d46d-43d7-af80-12a995ae527a">

### DBeaver Version

Version 23.2.0.202309041200

### Operating System

Mac, Ventura, M1 chip

### Database and driver

1. Driver for MySQL 8 and later. ID -mysql8.
2. I can't connect a database but I would connect MySQL

### Steps to reproduce

1. Click on 'create new database connection', but the button is grayed out.

**Comment by E1izabeth** [comment, marked as answer]
Could you try resetting perspective? Normally you should see [Navigator ](https://dbeaver.com/docs/dbeaver/Database-Navigator/) on the left.
![image](https://github.com/dbeaver/dbeaver/assets/28875055/3d2fcade-658d-46d2-8a96-e4b029acef27)

**Comment by fishmonger1** [comment]
Thank you very much Elizabeth, this solved the issue.
